word2vec.py

```diff
@@ -33,15 +38,16 @@ def __iter__(self):
                 if curr_pmid is not None and curr_total is not None:
                     if "Disease" in line or "Gene" in line:  # targeting gene-disease pairs
                         features = line.split("\t")
-                        if int(features[0]) == int(curr_pmid) and features[5].strip() != "":
+                        if features[0] == curr_pmid and features[5].strip() != "":
                             curr_total = curr_total.replace(features[3], features[5].strip(), 1)
         yield curr_total.split()
-                    
 
-def create_word2vec(sentences):
+
+def word2vec(sentences):
     """
     Creates a word2vec model.
-    @param sentences: list of list of words in each sentence (title + abstract).
+    @param sentences: list of list of words in each sentence (title + abstract)
```

Comment:
whoa awesome comments. Is this for a documentation string generator or do you just write comments this way?

Reply:
both! i believe PEP8 has more specific guidelines on how to structure docstrings, but i do what i can